tests/services/repo-creator.service.spec.ts

```diff
@@ -37,6 +41,32 @@ describe('RepoCreatorService', () => {
     });
   });
 
+  describe('.requestRepoCreationPermissions()', () => {
+    it('should not need to return any permissions if the repo was already created', () => {
+      of(true)
+        .pipe(repoCreatorService.requestRepoCreationPermissions(Phase.phaseBugReporting, PHASE_REPO))
+        .subscribe((repoCreationPermission: boolean | null) => expect(repoCreationPermission).toBe(null));
+    });
+
+    it('should not need to return any permissions if the repo was not created but current phase is not bugReporting', () => {
```

Comment:
Maybe something like this might be a little clearer? The `should not need to` part feels abit like the requirement is optional 😅 

```suggestion
    it('should not return any permissions if the repo was not created and the current phase is not bugReporting', () => {
```

Reply:
👍

Reply:
Yea >< it would be better to be more firm wrt to the test naming 😅